wallet/builder.go

```diff
@@ -7,12 +7,13 @@ import (
 
 	"github.com/ava-labs/avalanchego/wallet/subnet/primary"
 
+	"github.com/ava-labs/avalanche-tooling-sdk-go/account"
 	"github.com/ava-labs/avalanche-tooling-sdk-go/wallet/chains/pchain"
 	"github.com/ava-labs/avalanche-tooling-sdk-go/wallet/types"
 )
 
 // BuildTx constructs a transaction for the specified operation
-func BuildTx(wallet *primary.Wallet, params types.BuildTxParams) (types.BuildTxResult, error) {
+func BuildTx(wallet *primary.Wallet, account account.Account, params types.BuildTxParams) (types.BuildTxResult, error) {
```

Comment:
i dont know why you are adding accoutn here, buildtx works as is and is alrady clean. account is already part of params